src/scripting/abc.cpp

```diff
@@ -589,6 +591,7 @@ void ABCVm::registerClasses()
 	builtin->registerBuiltin("TextFormatAlign","flash.text",Class<TextFormatAlign>::getRef(m_sys));
 	builtin->registerBuiltin("TextLineMetrics","flash.text",Class<TextLineMetrics>::getRef(m_sys));
 	builtin->registerBuiltin("TextInteractionMode","flash.text",Class<TextInteractionMode>::getRef(m_sys));
+	builtin->registerBuiltin("TextRenderer","flash.text",Class<StaticText>::getRef(m_sys));
```

Comment:
This should be ...Class<TextRenderer>::getRef(...
that should also make your test case work

Reply:
Thanks for the help! Really appreciate it :smiley:. I'll take it slower with future PRs as quite a few mistakes were done in this one